evcouplings/couplings/model.py

```diff
@@ -5,15 +5,19 @@
 
 Authors:
   Thomas A. Hopf
+  Anna G. Green (MultiSegmentCouplingsModel)
 """
-
 from collections import Iterable
 from copy import deepcopy
 
 from numba import jit
 import numpy as np
 import pandas as pd
 
+from evcouplings.couplings.mapping import (
```

Comment:
I think this is your import ;) when removed, moving the class over to mapping.py should work